kernel/src/actions/deletion_vector.rs

```diff
@@ -55,11 +56,10 @@ impl DeletionVectorDescriptor {
         match self.storage_type.as_str() {
             "u" => {
                 let path_len = self.path_or_inline_dv.len();
-                if path_len < 20 {
-                    return Err(Error::deletion_vector(
-                        "Invalid length {path_len}, must be >20",
-                    ));
-                }
+                require!(
+                    path_len >= 20,
+                    Error::deletion_vector("Invalid length {path_len}, must be >= 20",)
```

Comment:
nit: I don't think we need that trailing comma? 

Also, I don't see anything like a `format!`, will the string actually get formatted?